connexta/eve
# 0911 dev to master

#### What does this PR do?
FINAL POWER MERGE FROM DEV TO MASTER.

#### Who is reviewing it?

@vinamartin @alchucam @mcalcote 

@Bdthomson @adimka @samuelechu @djblue @gordocanchola

#### How should this be tested?
Click on the last commit, Hide whitespace changes, double check if any code has been modified.

#### Screenshots


#### Checklist:

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
